src/test/config.sol

```diff
@@ -82,10 +82,10 @@ contract Config {
         bool    offboarding;
     }
 
-    uint256 constant HUNDRED    = 10 ** 2;
-    uint256 constant THOUSAND   = 10 ** 3;
-    uint256 constant MILLION    = 10 ** 6;
-    uint256 constant BILLION    = 10 ** 9;
+    uint256 constant         HUNDRED  = 10 ** 2;
```

Comment:
Optional reduction of constant reliance
```suggestion
```

Also requires `dust:         75 * HUNDRED,` in WBTC (Line 384) changing to `dust:         75_00,`

Which IMO is the way to go

Reply:
IMO `HUNDRED` doesn't really make much sense, so I would vote to remove it as well.

Reply:
Ya agreed to remove.